### DISCUSSION TITLE
Transfers not showing in report and category report

### DISCUSSION THREAD

**Discussion by dc3d** [firefly-iii/firefly-iii#9043] [Firefly III]
Hello,

Thanks so much for this software JC5, it's amazing so far!

I'm not sure if I'm going about this correctly or if there is another way:

I have an asset account called : my_checking_account
I have my bank called : SuperBank
And a shared account at this bank for my mortgage interests (withdrawn every 3 months) called : SuperBank_Shared_Mortgage_Account
Someone depositing on the shared account called : Julia

I have 4 categories : Mortgage_Me, Mortgage_Julia for deposit, and Mortgage and fees for expenses.
The only place I see my transfers is in income account, but not showing in categories which makes it hard to read, below the screenshot:
![Financial_report_2023_edited](https://github.com/user-attachments/assets/d3f7ce80-9cc1-47b9-94c7-0ac68ff126b7)
![Category_Report_2023_edited](https://github.com/user-attachments/assets/fd4e2511-8959-411a-8c5c-9a90823adacd)

and there is the info button related to the income in default financial report:
![My_Checking_Account-Mortgage_Me_edited](https://github.com/user-attachments/assets/9522332a-90f2-49cc-ad1a-47fe6603855e)

![Julia-Mortgage_Julia_edited](https://github.com/user-attachments/assets/41926ea0-e13b-4c42-b1a1-c801985e3755)

How can I see my transactions in the category row in the default financial report, and how can I see it in the category report, as well as average income per source account?

Thanks for your help, looking forward to it.

**Comment by JC5** [comment]
Transfers are excluded from these reports when both accounts are selected. Select just one of them to see the transfers as either expense or income.

**Reply by dc3d** [reply]
Hi, Thanks for your reply,
Yes, I should have mentioned that's what was the main issue I saw when reading all the discussions related to transfer and report.
Just to be correct, this is what you're referring to ?
![2024-07-13 23 29 55 demo firefly-iii org a947589eb4a8](https://github.com/user-attachments/assets/9b16c124-93f3-43d3-9caa-efe885135260)
If so, that's the case, I only have the shared asset mortgage account selected.
I also tried disabling the net worth as well as removing the active checkmark in the account without success.
I tried with the demo to add a category-test to the shared account transfer without success as well
![2024-07-13 23 57 12 demo firefly-iii org 9a521f626661](https://github.com/user-attachments/assets/cd3ae721-3e1b-4f33-ad30-d3e8c5d235eb)
![2024-07-13 23 57 23 demo firefly-iii org dde781656677](https://github.com/user-attachments/assets/975c03e2-797c-4323-a3e6-de61cf761d9a)
![2024-07-13 23 58 01 demo firefly-iii org a77c1de4d01b](https://github.com/user-attachments/assets/3ffec348-ab6a-47e1-8fc0-dd382d365985)

And finally on the demo account, I tried to create a shared account z_SuperBank_mortgage with transfer and deposit and payment
![2024-07-13 23 53 57 demo firefly-iii org 0498c4f87890](https://github.com/user-attachments/assets/31d271f8-d3b7-4dc3-9155-e93bfe285866)
![2024-07-13 23 51 48 demo firefly-iii org 4fcc8310b8d7](https://github.com/user-attachments/assets/1a3155f4-eb0d-4425-bdfc-1cddded3abd3)
![2024-07-13 23 52 33 demo firefly-iii org 02de8801b492](https://github.com/user-attachments/assets/69675e4c-ad54-415c-acea-52f58e1d1da9)
![2024-07-13 23 53 28 demo firefly-iii org dc8a7c0d13c1](https://github.com/user-attachments/assets/91894b8a-b030-4cbd-b2b3-35425bdd5366)

Maybe it only works with liability ? Or maybe I should do a withdrawal to a name similar from my bank account and then do a deposit from it so it's not treated as a transfer? But it'll get a lot more complicated.

I'm on version 6.1.18

Hopefully you can point me in the correct direction, thanks again for your support!
